db/updateDb.js

```diff
@@ -56,41 +56,44 @@ async function fillCasesByCountry(countryGeoId) {
 
     let countryRecords = await getByCountry(countryGeoId);
 
-    let db = new sqlite3.Database(common.serverConfig.db.database);
+    const db = new Database(common.serverConfig.db.database);
 
 
     console.log(' - filling cases for country ' + countryGeoId);
 
-    db.serialize(function () {
 
-        let stmt = db.prepare("INSERT INTO cases(country, date, cases, deaths) VALUES (?, ?, ?, ?)");
+    let stmt = db.prepare("INSERT INTO cases(country, date, cases, deaths) VALUES (?, ?, ?, ?)");
 
-        stmt.on('error', (err) => {
+    let rowsInserted = 0;
+    let lastInsertRowid;
 
+    for (let record of countryRecords) {
 
-        });
+        let year = record.year;
+        let month = parseInt(record.month) - 1;
+        let day = record.day;
 
-        for (let record of countryRecords) {
+        let date = (new Date(Date.UTC(year, month, day))).toJSON();
 
-            let year = record.year;
-            let month = parseInt(record.month) - 1;
-            let day = record.day;
 
-            let date = (new Date(Date.UTC(year, month, day))).toJSON();
+        try {
 
+            const info = stmt.run(countryGeoId, date, record.cases, record.deaths);
 
-            stmt.run(countryGeoId, date, record.cases, record.deaths);
+            rowsInserted += info.changes;
+            lastInsertRowid = info.lastInsertRowid;
+
+        } catch (err) {
 
```

Comment:
Maybe put somthing here, good idea to fill this kind og blank

When trying, an error is resulted: SqliteError: ```FOREIGN KEY constraint failed```
It occures because:
 - The update script now acts as a module so when executed (like before) it does not create the FR country (method ```addCountry``` is not used by the way)
 - So when updating a foreign key error is thrown

Advice:
 - Reshape de mehanic of this system to auto add countries when recieving new country case data, do not rely on user/dev adding manualy the countries